lib/shoulda/matchers/action_controller/permit_matcher.rb

```diff
@@ -38,7 +38,16 @@ module ActionController
       #     describe UsersController do
       #       it do
       #         should permit(:first_name, :last_name, :email, :password).
-      #           for(:create).
+      #           for(:create,
+      #             params: {
+      #               user: {
+      #                 first_name: 'Jon',
+      #                 last_name: 'Doe',
+      #                 email: 'jondoe@example.com',
+      #                 password: 'password'
+      #               }
+      #             }
+      #           ).
       #           on(:user)
```

Comment:
Oof I don't like how this forces you to write it this way. I'd probably write it this way in Real Code:

``` ruby
it do
  params = {
    user: {
      first_name: "Jon",
      last_name: "Doe",
      email: "jondoe@example.com",
      password: "password"
    }
  }
  should permit(:first_name, :last_name, :email, :password).
    for(:create, params: params).
    on(:user)
end
```

But, this is fine.